baselines/dom.generated.d.ts

```diff
@@ -18878,6 +18840,29 @@ declare var webkitRTCPeerConnection: {
 
 declare type EventListenerOrEventListenerObject = EventListener | EventListenerObject;
 
+declare namespace console {
```

Comment:
I'll re-test @mhegazy's question. Here's how I'll do it in case you want to try it yourself.

```sh
$ git checkout console
$ npm test
$ cp baselines/* ~/ts/src/lib
$ cd ~/ts
$ gulp
$ cd ~/dtslint-runner
$ npm run build
$ node bin/index.js --localTs ~/ts/built/local
```

You need clones of Typescript, dtslint-runner and DefinitelyTyped to make this work.